k8s/upgrades/README.md

```diff
@@ -93,13 +93,13 @@ latest stable/openebs chart.
 - If the default values seem appropriate, you can use the below commands to
   update OpenEBS. [More](https://hub.helm.sh/charts/stable/openebs) details about the specific chart version.
   ```sh
-  $ helm upgrade --reset-values <release name> stable/openebs --version 1.11.0
+  $ helm upgrade --reset-values <release name> stable/openebs --version 1.12.0
   ```
 - If not, customize the values into your copy (say custom-values.yaml),
   by copying the content from above default yamls and edit the values to
   suite your environment. You can upgrade using your custom values using:
   ```sh
-  $ helm upgrade <release name> stable/openebs --version 1.11.0 -f custom-values.yaml`
+  $ helm upgrade <release name> stable/openebs --version 1.12.0 -f custom-values.yaml`
```

Comment:
this commands need to be verified. The new command could be: 
```
helm repo update
helm upgrade <release name> openebs/openebs --version 1.12.0
```

Reply:
Verified, works for stable/openebs to openebs/openebs charts.